r128gain/__init__.py

```diff
@@ -340,6 +341,13 @@ def tag(filepath, loudness, peak, *,
     return
 
   mf.save()
+  # preserve original modification time, possibly increasing by some seconds
+  if add_seconds is not None:
+    if add_seconds == 0:
+      logger().info("Resetting modification time for file '{}'".format(filepath))
```

Comment:
Wording: "restoring" would be more accurate than "resetting".
Also use debug logging level for this.

Reply:
Totally agree re wording.

As for log level: I grepped for all logging in __init__.py and thought this would fit better with your info than with your debug log entries. If you feel different, I'll change it of course.